packages/block-library/src/term-name/block.json

```
@@ -0,0 +1,68 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "core/term-name",
```

```suggestion
	"apiVersion": 3,
	"__experimental": true,
	"name": "core/term-name",
```

Without the `__experimental` field, there is a risk that this experimental block will be shipped to the core.

---

Thanks - accidentally missed adding this.